lib/Dialect/TritonGPU/Transforms/Pipeliner/LowerLoops.cpp

```diff
@@ -504,8 +504,8 @@ scf::ForOp lowerLoads(scf::ForOp forOp, CoarseSchedule &schedule) {
           // Allocate additional buffer required by the wgmma pipelining.
           stageDiff += 1;
         }
-        asyncLoads[&op] = {.stageDiff = stageDiff,
-                           .sharedEncoding = sharedEncoding};
+        asyncLoads[&op].stageDiff = stageDiff;
+        asyncLoads[&op].sharedEncoding = sharedEncoding;
```

Comment:
NIT: It's a bit inefficient to do the map lookup twice

Reply:
Agree